src/site/resources/js/checkstyle.js

```diff
@@ -58,6 +58,16 @@ window.addEventListener("load", function () {
     externalLinks.forEach((link) => {
         link.setAttribute("target", "_blank");
     });
+
+    const codeBlocks = document.querySelectorAll(".prettyprint code");
+    codeBlocks.forEach((block) => {
+        const code = block.innerText.split("\n");
+        if (code.length > 1) {
+            code.shift();
+            code.pop();
```

Comment:
before removal, of first and last lets check that they have no printed characters to make sure no meaningful lines are removed.